modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/context/SelectAccountUsersManagementToolbarDisplayContext.java

```diff
@@ -53,6 +59,46 @@ public String getClearResultsURL() {
 		return clearResultsURL.toString();
 	}
 
+	@Override
+	public CreationMenu getCreationMenu() {
+		if (!isSingleSelect()) {
+			return null;
+		}
```

Comment:
We can remove this since we do want to show the creation menu for https://issues.liferay.com/browse/LPS-115260.

Reply:
@pei-jung agreed, though I have not tested it at all. If there are any issues with the behavior of the plus button in that context, I'd like to address them in that ticket. This early return is intended to simplify the scope of this ticket.

Reply:
👍